includes/fields/class-fieldtypes-select.php

```diff
@@ -251,23 +251,35 @@ private function add_options_to_field( $options, $value, $field ) {
 			$option_data = array(
 				'label'      => $label,
 				'attributes' => array(),
+				'value'      => esc_attr( $option ),
```

Comment:
Based on the WordPress Handbook (https://developer.wordpress.org/apis/security/escaping/#toc_3), we should always escape late. So, we don’t need `esc_attr` here.

I’ve checked, and we’re already escaping this value correctly later. We just need to remove the `esc_attr` here.